lib/components/Modal/bottom/index.tsx

```diff
@@ -0,0 +1,160 @@
+import ReactDOM from "react-dom";
+import { useState, useEffect, useRef } from "react";
+import clsx from "clsx";
+import "./modal-bottom.scss";
+import { useSwipeable } from "react-swipeable";
+import { ModalHeader } from "./modal-header";
+import { ModalBody } from "./modal-body";
+import { Button } from "../../Button";
+
+interface ModalBottomProps {
```

Comment:
interface ModalBottomProps extends HTMLAttributes<HTMLDivElement>

Reply:
I think that it's necessary when you are working with buttons, inputs and etc, but div is not so specific. Probably, it might be case-by-case, but I'll apply in order not to restrict future user of the library